app/src/main/java/org/schabi/newpipe/local/feed/notifications/NotificationIcon.java

```diff
@@ -0,0 +1,60 @@
+package org.schabi.newpipe.local.feed.notifications;
```

Comment:
Is there a specific reason you did not create this file as Kotlin like all of the others related to feed?

Reply:
done